src/core/main.go

```diff
@@ -251,7 +252,7 @@ func main() {
 	beego.InsertFilter("/*", beego.BeforeRouter, filter.SecurityFilter)
 	beego.InsertFilter("/*", beego.BeforeRouter, filter.ReadonlyFilter)
```

Comment:
TODO REMOVE?

Reply:
Issue https://github.com/goharbor/harbor/issues/10532 opened to track the work